config/jobs/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/config.yaml

```diff
@@ -68,3 +68,19 @@ presubmits:
         - "--scenario=execute"
         - "--" # end bootstrap args, scenario args below
         - "hack/verify-all.sh"
+  - name: pull-gcp-compute-persistent-disk-csi-driver-kubernetes-integration
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--root=/go/src"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--clean"
+        - "--timeout=1800"
```

Comment:
is this in minutes or seconds?

Reply:
seconds, this is 30 minutes

Reply:
minutes 🤷‍♂️

Reply:
wot. Then its not doing anything.... now that I do the math it seems weird. My e2e config has `--timeout=900` and i've observed 10 minute timeouts. Didn't think about it till now but 900seconds is not even 10 minutes :S